testing/i3blocks/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Contributor: Marvin Steadfast <marvin@xsteadfastx.org>
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
+pkgname=i3blocks
+pkgver=20190321_git
+pkgrel=0
+pkgdesc="A minimalist scheduler for your status line scripts"
+url="https://github.com/vivien/i3blocks"
+arch="all"
+license="GPL"
+depends=""
+makedepends="autoconf automake"
+install=""
+subpackages="$pkgname-doc"
+_commit="ec050e79ad8489a6f8deb37d4c20ab10729c25c3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/vivien/i3blocks/archive/$_commit.tar.gz"
+builddir="$srcdir/$pkgname-$_commit"
+
+build() {
+	cd "$builddir"
+	./autogen.sh
+	./configure
+	make
+}
+
+check() {
+	cd "$builddir"
```

Comment:
it should do a check, is there `make check`

Reply:
done

Reply:
there is no check make target, please remove `check()`